### ISSUE TITLE
StompInNode requires JSON payload 

### ISSUE THREAD

**Issue by mfeblowitz** [node-red/node-red#618]
The message body (payload) received via a subscription might not be parseable JSON. If what comes through is not parseable JSON, an exception halts node-red (triggered at line 61)

It seems that payloads should generally be allowed to be either parseable JSON or non-JSON strings, and handled accordingly, without crashing node-red.

**Comment by knolleary** [comment]
Err, yes absolutely. No idea why its doing this - it should be passing on the received message as-is.

Although, given we've shipped this behaviour, we may have to stick with the auto parsing of JSON, but pass on the original message if the parse fails. Although, maybe we can just break it.... will be worth looking at the npm install stats to get a feel for how used the node is.

By the way, the Stomp node is actually in this repository: https://github.com/node-red/node-red-nodes so issues for it should be raised over there - no need to move this one now its raised.

**Comment by mfeblowitz** [comment]
Ah - still learning :)

I'm experimenting with this code as a local fix for now:

```
    console.log("===>>> Message received");
        try {
                msg.payload = JSON.parse(body);
            } catch (e) {
            msg.payload = body;
           }
```

rather than 

```
    console.log("===>>> Message received");
    msg.payload = JSON.parse(body);
```

**Comment by dceejay** [comment]
Looks good to me.

**Comment by dceejay** [comment]
Hi Mark, I have published a fix to npm - including more docs and the fix above.
Let me know if we are getting there.

**Comment by dceejay** [comment]
Closed by fix to node-red-nodes - 31318b03d769c388490f97eccca52b5850ef1261

**Comment by mfeblowitz** [comment]
Most definitely (and swiftly).

The changes look good.

Thanks!

Mark
